src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+
+import org.junit.jupiter.api.Test;
+
+public class DukeTest {
+
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+    private final PrintStream originalOut = System.out;
+    private final PrintStream originalErr = System.err;
+
+    public void setUpStreams() {
```

Comment:
This is a very useful way to set up tests! I have not tried it myself but I think there may be a way to automate it, rather than doing it manually in each of the test cases. You can try adding `@BeforeEach` and `@AfterEach`. If you tried and it works or does not work, please feel free to let me know!